android/fastlane/Fastfile

```diff
@@ -126,4 +126,177 @@ end
       skip_upload_apk: true,
     )
   end
+  desc "Deploy APK to Firebase App Distribution"
```

Comment:
I am not verify familiar with the best practices from fastlane but having big block of code in config file is hard to maintain, make changes over time and test it.

In an ideal world there should be a script against which we can write some tests and that script should be invokable from here

Reply:
Gentle reminder to address this comment.